test/unit/warp/src/Activemask.cpp

```diff
@@ -83,13 +83,13 @@ class ActivemaskMultipleThreadWarpTestKernel
 TEMPLATE_LIST_TEST_CASE( "activemask", "[warp]", alpaka::test::acc::TestAccs)
 {
     using Acc = TestType;
-    using Dev = alpaka::dev::Dev<Acc>;
-    using Pltf = alpaka::pltf::Pltf<Dev>;
+    using Dev = alpaka::Dev<Acc>;
+    using Pltf = alpaka::Pltf<Dev>;
     using Dim = alpaka::dim::Dim<Acc>;
     using Idx = alpaka::idx::Idx<Acc>;
 
-    Dev const dev(alpaka::pltf::getDevByIdx<Pltf>(0u));
-    auto const warpExtent = alpaka::dev::getWarpSize(dev);
+    Dev const dev(alpaka::getDevByIdx<Pltf>(0u));
+    auto const warpExtent = alpaka::getWarpSize(dev);
```

Comment:
Just as a note. The changed version looks visually better to me, and looking through the changed part it mostly never became confusing. This line however may be.

Reply:
Hmm. I wonder how this can be confusing? I call `getWarpSize` and I pass in a `Dev` called `dev`. Of course I need to know what a warp is. But how could it not be clear, that I query a warp size of a device?

Reply:
To me, it is a subjective mismatch that warp is a really specific and low-level thing and feels weird to access it "directly" via `alpaka::` first-level namespace.

Reply:
I think I understand your concern, but I also do not think that separating lower-level functionality into different namespaces is a solution. I also cannot recall any C++ library that does that. I am also worried that deciding what is low level and what not might be subjective as well. I think **if** sub namespaces are used, then they should be used to separate domains (cf. std::filesystem, std::chrono or std::ranges).

Reply:
>  I think if sub namespaces are used, then they should be used to separate domains (cf. std::filesystem, std::chrono or std::ranges).

At least from my perspective, this is exactly what is done in alpaka. The only difference is the granularity.

Reply:
Well, to some degree yes. Alpaka is trying to group the right things together. But the point is that there are far too many namespaces. We discussed this in length in VCs and in #1034. This is also the feedback we get from e.g. CERN.

And also in the standard library, `std::filesystem` is already so long, that you can frequently see an alias like `namespace fs = std::filesystem` in many examples on the web.